Sources/muterCore/TestReporting/TestSuiteOutcome.swift

```diff
@@ -52,6 +61,12 @@ extension TestSuiteOutcome {
         return testLog.contains("xcodebuild: error:") ||
             testLog.contains("error: terminated") ||
             testLog.contains("failed with a nonzero exit code") ||
-            testLog.contains("Testing cancelled because the build failed")
+            testLog.contains("Testing cancelled because the build failed") ||
+            testLog.contains("Command failed with exit code 1.")
     }
 }
+
+private enum BinaryType {
```

Comment:
This is a clue that we want some sort of "closed", loop-based solution to me. Check out how we generate configuration files as an example - it either parses out a config or returns nil. I'm thinking we want to take that approach here, though to do that we'll need to refactor the existing code to return nil or the outcome.

https://github.com/SeanROlszewski/muter/blob/master/Sources/muterCore/Configuration/configurationGeneration.swift